app/Datmusic/DownloaderTrait.php

```diff
@@ -296,13 +296,13 @@ public function downloadFile($url, $path)
         curl_setopt($curl, CURLOPT_CONNECTTIMEOUT, config('app.downloading.timeout.connection'));
         curl_setopt($curl, CURLOPT_TIMEOUT, config('app.downloading.timeout.execution'));
 
-        if (config('app.proxy.enabled')) {
-            curl_setopt($curl, CURLOPT_PROXY, config('app.proxy.ip'));
-            curl_setopt($curl, CURLOPT_PROXYPORT, config('app.proxy.port'));
-            curl_setopt($curl, CURLOPT_PROXYTYPE, config('app.proxy.method'));
+        if (env('PROXY_ENABLE', false)) {
+            curl_setopt($curl, CURLOPT_PROXY, env('PROXY_IP', null));
```

Comment:
Change `env('PROXY_IP', null)` to `env('PROXY_IP')`. Use fallback only if it's boolean like `PROXY_ENABLED`.